.gitignore

```diff
@@ -6,8 +6,6 @@ node_modules
 .eslintcache
 .npmrc
 .yarn-error.log
-.nyc_output
 !.github/
```

Comment:
This could probably be removed too since it's moot because the folder is committed.